src/main.ts

```diff
@@ -12,6 +12,14 @@ export const config = {
     type: 'boolean',
     default: false,
   },
+  replaceText: {
+    type: 'boolean',
+    default: false,
+  },
+  appendFileName: {
+    type: 'boolean',
+    default: true,
```

Comment:
Defaults are the opposite of what I would expect. The name and description of the package both suggest that it works as a unix filter, i.e. takes input on stdin and produces output on stdout. Neither passing additional arguments nor ignoring stdout should be the default.

Reply:
Yep, the defaults in final PR should be the opposite.